platform/viewer/.webpack/writePluginImportsFile.js

```diff
@@ -58,34 +60,87 @@ function getFormattedWindowBlock(addToWindowLines) {
 }
 
 function getRuntimeLoadModesExtensions() {
-  return "\n\n// Add a dynamic runtime loader\n" +
-    "export default async () => {\n" +
-    " for(const modeFactory of modesFactory) {\n" +
-    "  const newModes = await modeFactory(modes,extensions);\n" +
-    "  newModes.forEach(newMode => modes.push(newMode));\n" +
-    "}\n}\n";
+  return (
+    '\n\n// Add a dynamic runtime loader\n' +
+    'export default async () => {\n' +
+    ' for(const modeFactory of modesFactory) {\n' +
+    '  const newModes = await modeFactory(modes,extensions);\n' +
+    '  newModes.forEach(newMode => modes.push(newMode));\n' +
+    '}\n}\n'
+  );
 }
 
-const createCopyPluginFromExtensions = (SRC_DIR, DIST_DIR, plugins) => {
-
-  return plugins.map(plugin => {
-    const from = `${SRC_DIR}/../node_modules/${plugin.packageName}/public/`;
-    const exists = fs.existsSync(from);
-    return exists ? {
-      from,
-      to: DIST_DIR,
-      toType: 'dir',
-    } : undefined;
+const createCopyPluginToDistForLink = (
+  SRC_DIR,
+  DIST_DIR,
+  plugins,
+  folderName
+) => {
+  return plugins
+    .map(plugin => {
+      const from = `${SRC_DIR}/../node_modules/${plugin.packageName}/${folderName}/`;
+      const exists = fs.existsSync(from);
+      return exists
+        ? {
+            from,
+            to: DIST_DIR,
+            toType: 'dir',
+          }
+        : undefined;
+    })
+    .filter(x => !!x);
+};
+
+const createCopyPluginToDistForBuild = (
+  SRC_DIR,
+  DIST_DIR,
+  plugins,
+  folderName
+) => {
+  return plugins
+    .map(plugin => {
+      const from = `${SRC_DIR}/../../../node_modules/${plugin.packageName}/${folderName}/`;
+      const exists = fs.existsSync(from);
+      return exists
+        ? {
+            from,
+            to: DIST_DIR,
+            toType: 'dir',
+          }
+        : undefined;
+    })
+    .filter(x => !!x);
+};
+
+const createCopyPluginChunksToDistForBuild = (SRC_DIR, DIST_DIR, plugins) => {
```

Comment:
This entire method and the calls to it are not required.  On windows, the way it ends up working causes the from file to not contain directory separators, which is mostly just a no-op, but occasionally it causes things to map to the wrong dist file, which means the dist value isn't available.  I suspect it works consistently in Linux.